packages/blade/src/components/AutoComplete/_decisions/decisions.md

```diff
@@ -0,0 +1,248 @@
+# AutoComplete
+
+A SelectInput where you can type inside the input to filter through the items.
+
+<img width="215" alt="image" src="https://github.com/razorpay/blade/assets/30949385/63d95ab7-6231-4986-befb-1634269e7af3">
+
+## Design
+
+- [Figma - SelectInput & AutoComplete](https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?node-id=13590%3A171038&mode=dev)
+
+## Props
+
+> **Note**
+>
+> Below this section, I have added [examples of common AutoComplete usage](#usage). Check them out to understand usage of these props in details.
+
+| **Props**          | **Description**                                                                                                      | **Type**                                                        | **Default Value** |
+| ------------------ | -------------------------------------------------------------------------------------------------------------------- | --------------------------------------------------------------- | ----------------- |
+| label              | label of input                                                                                                       | string                                                          |                   |
+| accessibilityLabel | aria-label of input when label is not defined                                                                        | string                                                          |                   |
+| labelPosition      | Label Position like other inputs but with new option `inside-input`                                                  | 'top' \| 'left' \| 'inside-input'                               | 'top'             |
+| maxRows            | height restrictions of input ([Checkout "Inactive - Active States" Usage](#with-different-inactive---active-states)) | 'single' \| 'multiple' \| 'expandable'                          | '3'               |
```

Comment:
```suggestion
| maxRows            | height restrictions of input ([Checkout "Inactive - Active States" Usage](#with-different-inactive---active-states)) | 'single' \| 'multiple' \| 'expandable'                          | 'single'               |
```

Reply:
or expandable depends what we decide

Reply:
I think 'multiple` might be better default since it has the best UX among these 3. Only place where multiple can't use is in header where we cant have layout shifts. There people will realize its breaking UI and use single anyway.

I'll check with Abhishek also once